js/lykke2.js

```diff
@@ -0,0 +1,1147 @@
+                    '2007': InsufficientFunds,
+                    '2008': InsufficientFunds,
+                    '2009': ExchangeError,
+                    '2010': InsufficientFunds,
+                    '2011': InvalidOrder,
+                    '2012': InvalidOrder,
+                    '2013': InvalidOrder,
+                    '2014': InvalidOrder,
+                    '2015': InvalidOrder,
+                    '2016': InvalidOrder,
+                    '2017': InvalidOrder,
+                    '2018': InvalidOrder,
+                    '2019': InvalidOrder,
+                    '2020': InvalidOrder,
+                    '2021': InvalidOrder,
+                    '2022': InvalidOrder,
+                    '2023': ExchangeError,
+                },
+                'broad': {},
+            },
+            'commonCurrencies': {
+            },
+        });
+    }
+
+    async fetchCurrencies (params = {}) {
+        const response = await this.publicGetAssets (params);
+        const currencies = this.safeValue (response, 'payload', []);
+        //
+        //     {
+        //         "payload":[
+        //             {
+        //                 "assetId":"115a60c2-0da1-40f9-a7f2-41da723b9074",
+        //                 "name":"Monaco Token",
+        //                 "symbol":"MCO",
+        //                 "accuracy":6,
+        //                 "multiplierPower":8,
+        //                 "assetAddress":"",
+        //                 "blockchainIntegrationLayerId":"",
+        //                 "blockchain":"ethereum",
+        //                 "type":"erc20Token",
+        //                 "isTradable":true,
+        //                 "isTrusted":true,
+        //                 "kycNeeded":false,
+        //                 "blockchainWithdrawal":true,
+        //                 "cashoutMinimalAmount":0.1,
+        //                 "lowVolumeAmount":null,
+        //                 "lykkeEntityId":"LYKKE NL",
+        //                 "siriusAssetId":0,
+        //                 "siriusBlockchainId":null,
+        //                 "blockchainIntegrationType":"none",
+        //                 "blockchainDepositEnabled":false,
+        //                 "isDisabled":false
+        //             }
+        //         ],
+        //         "error":null
+        //     }
+        //
+        const result = {};
+        for (let i = 0; i < currencies.length; i++) {
+            const currency = currencies[i];
+            const id = this.safeString (currency, 'assetId');
+            const code = this.safeString (currency, 'symbol');
+            const name = this.safeString (currency, 'name');
+            const type = this.safeString (currency, 'type');
+            const deposit = this.safeValue (currency, 'blockchainDepositEnabled');
+            const withdraw = this.safeValue (currency, 'blockchainWithdrawal');
+            const active = this.safeValue (currency, 'isDisabled');
```

Comment:
you need to negate this, so active = !isDisabled

Reply:
updated